k8s/stage/ingress/cert-manager.yaml

```diff
@@ -0,0 +1,6406 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
These manifests look good to me, Obviously I didn't read all the content of the CRDs but I checked deployments, roles, rolebindigs...
However, they are creating cluster level components, creating a new 'cert-manager' namespace and adding services to kube-system namespace. Because of that, I think this file should live in the cluster infra repo, together with secret-manager, metrics-server... https://github.com/mozilla-it/itsre-apps-infra/tree/master/k8s/flux/cluster